### PR TITLE
creare galerie si recenzii

### PR DESCRIPTION
**Pull request by roxananecsoi**

### CODE REVIEW

File: src/sass/index.scss
Comment (madalinagomei) [inline src/sass/index.scss:1]: @import './common'; - trebuie sa ramana

**Comment by madalinagomei** [comment]
nice done

**Comment by roxananecsoi** [comment]
Multumesc! :*

**Comment by madalinagomei** [comment]
if you have time, please make the home svg and the animation for customer

**Comment by roxananecsoi** [comment]
Yes leader! :))

**Review by madalinagomei — COMMENTED** [review]
ok

File: src/sass/index.scss
Comment (roxananecsoi) [reply src/sass/index.scss:1]: am adaugat!

**Pull request merged**